pyproject.toml

```diff
@@ -50,13 +50,7 @@ dependencies = [
     "pyflakes >= 2.1.1",
 ]
 
-[project.urls]
-"Homepage" = "https://github.com/PyCQA/flake8-pyi"
-"Source" = "https://github.com/PyCQA/flake8-pyi"
-"Bug Tracker" = "https://github.com/PyCQA/flake8-pyi/issues"
-"Changelog" = "https://github.com/PyCQA/flake8-pyi/blob/main/CHANGELOG.md"
-
-[project.optional-dependencies]
+[dependency-groups]
 dev = [
```

Comment:
We could split these up into multiple groups, but uv installs them all so fast that I don't think it's worth the additional complexity. (If we're going to make changes in that area, I'd also probably just remove black/flake8-bugbear/flake8-noqa/isort/pre-commit hooks, since we do our linting via pre-commit anyway).